dynet/cfsm-builder.cc

```diff
@@ -233,6 +233,15 @@ void ClassFactoredSoftmaxBuilder::serialize(Archive& ar, const unsigned int) {
   ar & p_rc2ws;
   ar & p_rcwbiases;
 }
+
+
+void ClassFactoredSoftmaxBuilder::initialize_expressions() {
```

Comment:
I could assume no parameters have been initialized yet, loop through them, and initialize all of them?  If I don't have that assumption, I basically am reimplementing get_rc2w and get_rc2wbias, or overwriting previously initialized expressions (which shouldn't be problematic?  not sure, though).